src/components/html/EcAdminSettingsForm.html

```diff
@@ -84,25 +96,28 @@
                       </samp>
                     </a>
 
-                    <div slot="description">
+                    <div 
+                      slot="description" 
+                      class="text-muted"
+                    >
                       {{ getObjectValues(item) }}
-                      <div class="mt-2" v-if="dataListsIndexes[field] === index">
-                        <a-button
-                          type="danger"
-                          size="small"
-                          html-type="button"
+                      <div 
+                        class="mt-2" 
+                        v-if="dataListsIndexes[field] === index"
+                      >
+                        <button 
+                          class="btn btn-danger rounded btn-xs"
                           @click.prevent="removeDataListElement(dataList, index, field)"
                         >
                           {{ i19delete }}
-                        </a-button>
+                        </button>
                       </div>
                     </div>
-                  </a-list-item-meta>
-                </a-list-item>
-              </a-list>
+                  </div>
+                </div>
+              </div>
             </div>
           </template>
-
           <div :key="formResetKey">
             <component
               :is="checkComplexSchema(schema) && field === config.field ? 'div' : 'a-form-item'"
```

Comment:
```suggestion
              :is="checkComplexSchema(schema) && field === config.field ? 'div' : 'a-form-item'"
```

Não usamos `a-form-item` mais, certo? Aqui deve ficar só `div` mesmo eu acho, talvez nem precisa mais ser um `<component>` dinâmico.